crates/oxc_transformer/src/react_jsx/options.rs

```diff
@@ -56,7 +55,7 @@ fn default_pragma_frag() -> Cow<'static, str> {
 impl Default for ReactJsxOptions {
     fn default() -> Self {
         Self {
-            runtime: ReactJsxRuntime::Automatic,
+            runtime: None,
```

Comment:
Use `Option` instead of `ReactJsxRuntime`, it is used to fallback default runtime according to the related scenario.